Nodejs/Product/Nodejs/TypeScriptHelpers/TypeScriptCompile.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudioTools.Project;
+
+namespace Microsoft.NodejsTools.TypeScript
+{
+    public static class TypeScriptCompile
+    {
+        private static string CompilerExe = "tsc.exe";
```

Comment:
>private static string CompilerExe = "tsc.exe"; [](start = 8, length = 46)

Out of curiosity, is there a reason to call the exe, rather than running tsc.js with node.exe?